src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,6 +27,10 @@
 public class RobotContainer {
   private final DriveTrain m_driveTrain = new DriveTrain();
 
+
+  public static final XboxController xboxController = new XboxController(0);
+
+  public static final  PhotonVision camera = new PhotonVision();
   // The robot's subsystems and commands are defined here...
```

Comment:
Is this comment necessary? If not remove it.
Either way add a newline separating `camera` and `m_exampleSubsystem` declarations.

Reply:
Maybe also delete `m_exampleSubsystem`?